internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -360,6 +362,14 @@ func RestoreIrqBalanceConfig(irqBalanceConfigFile, irqBannedCPUConfigFile, irqSm
 		return nil
 	}
 	if !fileExists(irqBannedCPUConfigFile) {
+		// check if folder exists, if not create /etc/sysconfig
+		if !fileExists(EtcSysconfig) {
```

Comment:
I don't think we can use fileexists, as it checks the path is not a directory (which we expect it to be). Maybe just do
```
if _, err := os.Stat(filename); os.IsNotExist(err) {
...
}
```

Reply:
You're right. I oversaw the info.IsDir() method call.